MANIFEST.in

```diff
@@ -1,4 +1,14 @@
-include *.rst *.txt
+include *.rst *.txt LICENSE
+exclude .editorconfig
+exclude .gitchangelog.rc
+exclude .importanizerc
+exclude .pre-commit-config.yaml
+exclude .style.yapf
+exclude Makefile
+exclude docker-compose.yml
+exclude pyproject.toml
+exclude tox.ini
+recursive-include docs *
 recursive-include tests *
```

Comment:
any reason to include tests in release tar file? maybe exclude?

Reply:
no reason, we can exclude